src/Components/Web.JS/src/Boot.WebAssembly.ts

```diff
@@ -61,3 +63,19 @@ if (shouldAutoStart()) {
     }
   });
 }
+
+async function intiializeConfigAsync(platform: Platform, resourceLoader: WebAssemblyResourceLoader) : Promise<void> {
+  const configFiles = resourceLoader.readConfigFilesAsync();
+  const resolvedFiles = await Promise.all(configFiles.map(async c => {
+    const content = new Uint8Array(await c.contentPromise);
+    return { name: c.name, content: content};
+  }));
+
+
+  window['Blazor']._internal.getApplicationEnvironment = () => platform.toDotNetString(resourceLoader.applicationEnvironment);
+  window['Blazor']._internal.getConfig = (string: System_String) : Pointer | undefined => {
+    const fileName = platform.toJavaScriptString(string);
+    const resolvedFile = resolvedFiles.find(f => f.name == fileName);
```

Comment:
```suggestion
    const resolvedFile = resolvedFiles.find(f => f.name === fileName);
```

Reply:
Also I've commented elsewhere about whether we should match these names case-insensitively. If we should, this check should be updated.